include/swift/AST/Decl.h

```diff
@@ -1883,6 +1883,9 @@ class PatternBindingEntry {
   /// The initializer context used for this pattern binding entry.
   DeclContext *InitContext = nullptr;
 
+  /// The text of the initializer expression if deserialized from a module.
+  StringRef InitStringRepresentation;
```

Comment:
Bitpack nitpick: there are a *lot* of pattern binding entries in the program. Can we cram this in with the initializer expression, and move the flags to the other members?